src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
```

Comment:
You need to create span something like that
```
const liList = tree.querySelectorAll('li');

for (const li of liList) {
  const span = document.createElement('span');

  li.prepend(span);
  span.append(span.nextSibling);
}
```
and than in listener first of all find nextSibling for closest span and work with it
```
 const item = e.target.closest('span').nextSibling;